css/enhancedsteam.css

```diff
@@ -2536,6 +2536,11 @@ input[type=checkbox].es_dlc_selection:checked + label {
     margin-top:72px;
 }
 
+/* comment editor */
+.blotter_poststatus_avatar {
+	top:  auto !important;
```

Comment:
Using !important is a measure of last-resort. Is there really no more-specific selector you could use here to avoid it?

Reply:
I could make it more specific. I'll try that.